manager/controllers/baseline_systems.go

```diff
@@ -205,7 +205,7 @@ func BaselineSystemsListIDsHandler(c *gin.Context) {
 	if err != nil {
 		return // Error handled in method itself
 	}
-	var resp = IDsResponse{IDs: ids}
+	var resp = ids2IDsPlainResponse(ids)
```

Comment:
same as for advisoriesIDs, let's change `systemsIDs` to return `IDsPlainResponse`

Reply:
Fixed